psutil/net.go

```diff
@@ -75,13 +86,21 @@ func netIOCounters(nss []core.Namespace) ([]plugin.MetricType, error) {
 		return nil, err
 	}
 
+	// gather conntrack metric
+	conntrack, err := net.FilterCounters()
+	if err != nil {
+		// Conntrack probably not loaded into the kernel.
+		fmt.Printf("Requested netfilter statistics %s not found", err)
```

Comment:
only print error and continue? I just want to make sure the desired behavior. Do you like to return the error here?